.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -1,49 +1,46 @@
-  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
-  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.
+## Purpose of this PR
 
-### Contribution checklist (MS Employees Only):
-- [ ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
-- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
-- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)
+What's the purpose of this PR? Check all that apply. This is **mandatory**!
 
-If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).
+  - [ ] new service onboarding
+  - [ ] new API version
+  - [ ] update existing version for new feature
+  - [ ] update existing version to fix swagger quality issue in s360
+  - [ ] other, please clarify:
+    - _edit this with your clarification_
 
-### ARM API Review Checklist
+## Due diligence checklist
 
-> **Applicability**: :warning:
->
-> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
-> - Change to data plane APIs
-> - Adding new properties
-> - All removals
+To merge this PR, you **must** go through the following checklist and confirm you understood 
+and followed the instructions by checking all the boxes:
 
-Otherwise your PR may be subject to ARM review requirements. Complete the following:
-- [ ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
-  - Adding a new service
-  - Adding new API(s)
-  - Adding a new API version
-    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version). Note that this doesn't apply if you are trying to merge a PR that was previously in the private repository.
+- [ ] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data-plane related specifications.
```

Comment:
>[ ] I confirm thi [](http://example.com/codeflow?start=2&length=17)

One key thing we want folks to understand is the PR review flowchart on how label management is done to progress the PR, so def acknowledging that they have gone thru that link is key.

Reply:
If you are talking about [the flowchart here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/909/API-spec-PR-review?anchor=management-plane-api-review) then I am like 100% sure this will confuse people. I think we probably should create a guidance that focused only on the most important labels like `ARMReview`, `ARMChangesRequested`, `WaitForARMFeedback` etc. But let me do this in a follow up PR. This PR is large already.

Reply:
sounds good